burn-autodiff/src/ops/tensor.rs

```diff
@@ -435,6 +435,23 @@ impl<B: Backend> TensorOps<Self> for Autodiff<B> {
             .stateless(B::neg(tensor.primitive))
     }
 
+    fn recip<const D: usize>(tensor: FloatTensor<Self, D>) -> FloatTensor<Self, D> {
+        #[derive(Debug)]
+        struct Recip;
+
+        impl<B: Backend, const D: usize> Backward<B, D, 1> for Recip {
+            type State = ();
+
+            fn backward(self, ops: Ops<Self::State, 1>, grads: &mut Gradients) {
+                unary::<B, D, D, _>(ops.parents, ops.node, grads, |grad| B::recip(grad));
+            }
+        }
+
+        Recip
+            .prepare([tensor.node], [tensor.graph])
+            .stateless(B::recip(tensor.primitive))
+    }
+
```

Comment:
We need add a test for this addition as well. [Coverage](https://app.codecov.io/gh/burn-rs/burn/pull/953?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=burn-rs#8fe7f54d3cbdf02d57a7370528618e69-R437) is showing a test is missing.

Reply:
Yes, even worse, the calculated tensor was not correct - I think, I could fix it, however, I'm not sure, if my implementation is the best, simplest, etc...